integration-tests/pages/checkAnswersPage.js

```diff
@@ -4,6 +4,44 @@ const clickSubmit = () => cy.get('[data-submit]').click()
 
 export default () =>
   page('Check your answers before submitting the report', {
+    verifyInputs: () => {
+      cy.get('.govuk-grid-column-full > :nth-child(2) > :nth-child(2) > :nth-child(2)').contains('Norman Smith')
```

Comment:
As discussed - can move this to use data-qa attributes in a follow up, when you render date too
e.g: `data-qa='name'` ?

Reply:
ok